devops/actions/aws-ec2/aws-ec2.js

```diff
@@ -1,6 +1,6 @@
 const core   = require('@actions/core');
 const github = require('@actions/github');
-const AWS    = require('aws-sdk');
+const { EC2, waitUntilInstanceRunning } = require('@aws-sdk/client-ec2');
```

Comment:
Was you able to test these changes in your own repo or it can potentially break stuff?